packages/abap-deploy-config-inquirer/src/index.ts

```diff
@@ -24,7 +26,7 @@ import { reconcileAnswers } from './utils';
  * @returns the prompts used to provide input for abap deploy config generation and a reference to the answers object which will be populated with the user's responses once `inquirer.prompt` returns
  */
 async function getPrompts(
-    promptOptions?: AbapDeployConfigPromptOptions,
+    promptOptions: AbapDeployConfigPromptOptions,
```

Comment:
Prompt options should be eh optional and default behaviour should be usable?

Reply:
Updated